### PR TITLE
Feature/427 taetigkeit konsumieren

### PR DESCRIPTION
**Pull request by edgardogc02**

### CODE REVIEW

File: lib/einsatzplanung/domain_model/taetigkeit/taetigkeit.ex [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  """
+
+  @derive Jason.Encoder
+  defstruct [
+    :id,
+    :name
+  ]
+
+  def neue(taetigkeit_id, taetigkeit_name) do
+    %__MODULE__{
```
Comment (osmanperviz) [inline lib/einsatzplanung/domain_model/taetigkeit/taetigkeit.ex]: @self?

File: lib/einsatzplanung/taetigkeit_korrigiert/taetigkeit_korrigiert.ex [hunk 2]
```diff
@@ -0,0 +1,20 @@
+defmodule Einsatzplanung.Events.TaetigkeitKorrigiert do
+  @moduledoc """
+    Taetigkeit korrigiert event module
+  """
+  defstruct [:id, :name]
+
+  def fuer(%Einsatzplanung.Taetigkeit{} = taetigkeit) do
+    %__MODULE__{
```
Comment (osmanperviz) [inline lib/einsatzplanung/taetigkeit_korrigiert/taetigkeit_korrigiert.ex]: @self?

File: lib/einsatzplanung/taetigkeit_korrigiert/taetigkeit_korrigiert_service.ex [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  defp taetigkeit_korrigieren(
+         auftrag: auftrag,
+         taetigkeit_id: taetigkeit_id,
+         taetigkeit_name: taetigkeit_name,
+         metadata: metadata
+       ) do
+    taetigkeit = Taetigkeit.neue(taetigkeit_id, taetigkeit_name)
+
+    starte_transaction()
+    |> Multi.run(:taetigkeit_korrigiert, fn _repo, %{} ->
```
Comment (osmanperviz) [inline lib/einsatzplanung/taetigkeit_korrigiert/taetigkeit_korrigiert_service.ex]: split 2 into functions

File: lib/einsatzplanung/individueller_taetigkeitsprofilname_festgelegt/individueller_taetigkeitsprofilname_festgelegt_service.ex [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        )
+    end
+  end
+
+  defp individueller_taetigkeitsprofilname_festlegen(
+         auftrag: auftrag,
+         individueller_taetigkeitsprofilname: individueller_taetigkeitsprofilname,
+         metadata: metadata
+       ) do
+    starte_transaction()
```
[the remark this answers is not in the document]
Comment (edgardogc02) [reply lib/einsatzplanung/individueller_taetigkeitsprofilname_festgelegt/individueller_taetigkeitsprofilname_festgelegt_service.ex]: DONE

File: lib/einsatzplanung/domain_model/taetigkeit/taetigkeit.ex (hunk 1 above)
Comment (edgardogc02) [reply lib/einsatzplanung/domain_model/taetigkeit/taetigkeit.ex]: Done

File: lib/einsatzplanung/taetigkeit_korrigiert/taetigkeit_korrigiert.ex (hunk 2 above)
Comment (edgardogc02) [reply lib/einsatzplanung/taetigkeit_korrigiert/taetigkeit_korrigiert.ex]: I cant do it here :(

File: lib/einsatzplanung/taetigkeit_korrigiert/taetigkeit_korrigiert_service.ex (hunk 3 above)
Comment (edgardogc02) [reply lib/einsatzplanung/taetigkeit_korrigiert/taetigkeit_korrigiert_service.ex]: Done

File: lib/einsatzplanung/taetigkeit_korrigiert/taetigkeit_korrigiert.ex (hunk 2 above)
Comment (osmanperviz) [reply lib/einsatzplanung/taetigkeit_korrigiert/taetigkeit_korrigiert.ex]: Buuuu!!!

**Pull request merged**